cli/Cargo.toml

```diff
@@ -16,6 +16,11 @@ name = "deno"
 path = "main.rs"
 doc = false
 
+[[bin]]
+name = "deno_standalone"
```

Comment:
`denort` maybe? That's what it used to be called.

Reply:
maybe call it "denoy" ? i guess this name is never exposed to the user, so maybe it should favor clarity over brevity, but "denort" sounds like "deno real time" to me.

Reply:
"denoy" is probably even more confusing 😅 

we've already known this as "denort" in the past, it makes sense to just continue using that term internally.